src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -142,28 +165,49 @@ private enum Flags : byte
                 _name = _lazySourceName = memberName;
             }
 
-            if ((isAutoProperty && hasGetAccessor) || hasInitializer)
+            if (isAutoProperty || hasInitializer)
```

Comment:
>if (isAutoProperty || hasInitializer) [](http://example.com/codeflow?start=12&length=37)

The motivation for this change is not clear. Could you please elaborate?

Reply:
@AlekseyTs The motivation was to allow for set-only semi-auto properties. But since this also allowed semicolon-only set auto properties, I'll need to change that to something like `if ((isAutoProperty && (hasGetAccessor || hasSetWithBody)) || hasInitializer)`